index.html

```diff
@@ -155,18 +155,18 @@ <h2 id ="project-title">
 </div>
 </section>
          <!-- Desktop work section -->
-         <section class="desktop-work">
+         <section id ="my-work" class="desktop-work">
             <div class="work"> 
-                <img src="media/Deskktop Snapshoot Portfolio.png" class="work-image" alt="Desktop Snapshoot Portfolio"></img>
+                <img src="media/Deskktop Snapshoot Portfolio.png" class="work-image" alt="Tonic project snapshot"></img>
```

Comment:
- [ ] While you did a good job adding `alt` attribute for almost all the images :+1: I have noticed that some of the have a closing tag `</img>`, kindly remember that the `img` element is self-closing so you shouldn't use `</img>`